dwave/system/samplers/dwave_sampler.py

```diff
@@ -218,7 +226,7 @@ def properties(self):
     def parameters(self):
         """dict[str, list]: D-Wave solver parameters in the form of a dict, where keys are
         keyword parameters accepted by a SAPI query and values are lists of properties in
-        :attr:`.DWaveSampler.properties` for each key.
+        :attr:`.properties` for each key.
```

Comment:
I was deliberating on that but given that sphinx has not supported this for a long time and might not for the foreseeable future, in a choice between between formally better RST syntax and live rather than dead links for the users, I prefer the latter.

Reply:
Did sphinx drop support for this? I missed that.

Reply:
We spoke about this maybe a year ago, when we worked through issues of the integrated docs in the SDk, remember all the warnings of type `WARNING: more than one target found for cross-reference 'properties':`? So when you use inheritance, the `.properties` in `LeapHybridSampler` might link to properties in `TilingSampler` in the html build. At the time we were reluctant to hardcode the path but we can't keep sending users off to the wrong places when they click a reference link.

Reply:
Fair enough. Ok. Wish sphinx was a better tool